.github/workflows/ci.yml

```diff
@@ -130,6 +130,7 @@ jobs:
     strategy:
       matrix:
         extras:
+          - "nopillow"
```

Comment:
I think I don't understand the purpose of this change. How is this test run different than others?